### PR TITLE
chore: resetup wallabyjs for babel

### PR DESCRIPTION
**Pull request by maksimr**

### CODE REVIEW

File: lib/middleware/karma.js [hunk 1]
```diff
@@ -232,5 +232,14 @@ var createKarmaMiddleware = function (
...
 
+createKarmaMiddleware.$inject = [
+  'filesPromise',
+  'serveStaticFile',
+  'serveFile',
+  'injector',
+  'config.basePath',
+  'config.urlRoot'
+]
+
```
Comment (dignifiedquire) [inline lib/middleware/karma.js]: Why do we need this?

**Comment by dignifiedquire** [comment]
needs rebase onto `master`

File: lib/middleware/karma.js (hunk 1 above)
Comment (maksimr) [inline lib/middleware/karma.js]: @dignifiedquire wallaby can't parse special comment like   /* config.basePath */ in arguments of the function

Comment (dignifiedquire) [inline lib/middleware/karma.js]: makes sense lets drop the comments then, as with this we double define things

**Comment by maksimr** [comment]
@dignifiedquire done

**Pull request merged**